### PR TITLE
Os 2

### PR DESCRIPTION
**Pull request by prem-charan**
## Issue that this pull request solves

Closes: # (issue number)

### Brief description of what is fixed or changed
added announcement functionality and also added doctors and pathologist schedule
fixed bugs in code
resolved api issues

## Types of changes

_Put an `x` in the boxes that apply_

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [ ] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [ ] I have performed a self-review of my own code
-   [ ] I have created new branch for this pull request
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings
-   [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by prem-charan** [comment]
@ChaudharyRaman @varun7singh @prajjwalkapoor

File: FusionIIIT/applications/scholarships/migrations/0002_auto_20240218_0103.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.5 on 2024-02-18 01:03
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
```
Comment (VarunGitGood) [inline FusionIIIT/applications/scholarships/migrations/0002_auto_20240218_0103.py]: remove these migrations

**Review by VarunGitGood — CHANGES_REQUESTED** [review]
remove this

**Comment by prem-charan** [comment]
Demo Video link :
https://drive.google.com/file/d/16mVkWS9FiwSLyKdKvuR5vg7MWTIcdrIw/view?usp=drive_link

**Comment by prem-charan** [comment]
@varun7singh migrations removed

File: FusionIIIT/applications/programme_curriculum/migrations/0002_auto_20240218_0103.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.1.5 on 2024-02-18 01:03
+
+from django.db import migrations, models
```
Comment (VarunGitGood) [inline FusionIIIT/applications/programme_curriculum/migrations/0002_auto_20240218_0103.py]: Remove this file

**Review by VarunGitGood — CHANGES_REQUESTED** [review]

**Comment by prem-charan** [comment]
@varun7singh file removed

**Comment by VarunGitGood** [comment]
LGTM

**Pull request merged**